test/integration/puppetlabs/puppetserver/testutils.clj

```diff
@@ -125,6 +125,31 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;; Interacting with puppet code and catalogs
 
+(schema/defn ^:always-validate
+  module-dir :- File
```

Comment:
(nit) all the other tests have this on one line, any reason for the inconsistency here?